pkg/engine/hybrid/crawl.go

```diff
@@ -61,6 +61,9 @@ func (c *Crawler) navigateRequest(s *common.CrawlSession, request *navigation.Re
 			statucCodeText = http.StatusText(statusCode)
 		}
 		httpreq, _ := http.NewRequest(e.Request.Method, URL.String(), strings.NewReader(e.Request.PostData))
+		for k, v := range c.Headers {
+			httpreq.Header.Set(k, v)
+		}
```

Comment:
header supports works in standard mode . but will not work in headless since headers can only be added using `page` . i think we can remove this . the support for custom headers will be introduced in headless through https://github.com/projectdiscovery/katana/issues/414